tests/contrib/kafka/test_kafka.py

```diff
@@ -0,0 +1,152 @@
+import confluent_kafka
+import pytest
+import six
+
+from ddtrace import Pin
+from ddtrace import tracer as dd_tracer
+from ddtrace.contrib.kafka.patch import patch
+from ddtrace.contrib.kafka.patch import unpatch
+from ddtrace.filters import TraceFilter
+from tests.contrib.config import KAFKA_CONFIG
+from tests.utils import override_config
+
+
+TOPIC_NAME = "test_topic"
+GROUP_ID = "test_group"
+BOOTSTRAP_SERVERS = "localhost:{}".format(KAFKA_CONFIG["port"])
+KEY = "test_key"
+if six.PY3:
+    PAYLOAD = bytes("hueh hueh hueh", encoding="utf-8")
+else:
+    PAYLOAD = bytes("hueh hueh hueh")
+
+
+class KafkaConsumerPollFilter(TraceFilter):
```

Comment:
With this filter, are we ever testing that the poll spans look as expected?

Reply:
`confluent_kafka.Consumer.poll()` is often used in this pattern (see their own [example here](https://github.com/confluentinc/confluent-kafka-python/blob/97f08fe107d259eff6f9c281a61d92e204d2935d/examples/consumer.py#L93-L104)):
```python
while True:
    message = consumer.poll(1.0)
    if message:
        # do things
        break
```
Since consumers often poll repeatedly until they get a message, most `consumer.poll()` spans will not have a message associated with them, and the number of empty (message-wise) `poll()` spans is non-deterministic. To get around this in snapshot testing, I added a filter here to get rid of all `poll()` spans that don't have messages associated with them, so the snapshot should always test that there is a `producer.produce()` span and a `consumer.poll()` span that has a message.

Reply:
Thanks for the nice explanation! I got that part, I just wanted to make sure that we're testing that the empty poll spans are looking as expected somewhere?

Reply:
Good point, I'll see if I can add that as a test. Thanks for catching!